x-pack/plugins/apm/server/lib/observability_dashboard/get_service_count.ts

```diff
@@ -8,18 +8,21 @@ import { ProcessorEvent } from '../../../common/processor_event';
 import { rangeFilter } from '../../../common/utils/range_filter';
 import { SERVICE_NAME } from '../../../common/elasticsearch_fieldnames';
 import { Setup, SetupTimeRange } from '../helpers/setup_request';
+import { getProcessorEventForAggregatedTransactions } from '../helpers/aggregated_transactions/get_use_aggregated_transaction';
 
 export async function getServiceCount({
   setup,
+  useAggregatedTransactions,
 }: {
   setup: Setup & SetupTimeRange;
+  useAggregatedTransactions: boolean;
```

Comment:
nit: `use` prefix leads me to think of react hooks. Can you rewrite to `has` or `is` prefix?
```suggestion
  isAggregatedTransactionsEnabled: boolean;
```